src/storage/event_listener.cc

```diff
@@ -59,6 +59,79 @@ std::string CompressType2String(const rocksdb::CompressionType type) {
   return "unknown";
 }
 
+std::string CompactionReason2String(const rocksdb::CompactionReason compression_reason) {
```

Comment:
We can use the same style as above. Put strings in `std::vector<std::string>` in order.